core-java-modules/core-java/src/test/java/com/baeldung/iterationcounter/ForLoopIterationCounterUnitTest.java

```diff
@@ -0,0 +1,48 @@
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import com.baeldung.iterationcounter.ForLoopIterationCounter;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ForLoopIterationCounterUnitTest {
+    private static final List<String> COLORS = Arrays.asList("red", "blue", "yellow", "green");
+
+
+    @Mock
+    private ForLoopIterationCounter iterationCounter = new ForLoopIterationCounter();;
```

Comment:
This seems confused. Two `;`s.

Is it a mock? If so, why are you `new`ing it? Why are you mocking the class under test?

Reply:
I needed a mock value for the last useless test, but I'll change it once I figure out how to test that lambda.